packages/manager/apps/pci-kubernetes/src/pages/new/hooks/useCusterCreationStepper.ts

```diff
@@ -4,12 +4,13 @@ import { TLocalisation } from '@ovh-ux/manager-pci-common';
 
 import { NodePoolPrice } from '@/api/data/kubernetes';
 import { TClusterPlan, UpdatePolicy } from '@/types';
+import { TLocation } from '@/types/region';
```

Comment:
Remark: it's weird to have an import '@/types' and a '@/types/region'. You should either have `import { TClusterPlan, TLocation, UpdatePolicy } from '@/types';`on each of them with their corresponding sub-folder.